src/main.js

```diff
@@ -1,18 +1,55 @@
-const prevButton = document.querySelector('.carousel__buttonPrev--js');
-const nextButton = document.querySelector('.carousel__buttonNext--js');
-const opinionBox = document.querySelector('.carousel__opinions--js');
-const carouselBox = document.querySelector('.carousel__box--js');
+const prevButton = document.querySelector('.js-carousel-prev');
+const nextButton = document.querySelector('.js-carousel-next');
+const opinionBox = document.querySelector('.js-carousel-opinions');
+const carouselBox = document.querySelector('.js-carousel-box');
+const screenBox = document.querySelector('.js-screen-box');
+const hamburger = document.querySelector('.js-hamburger');
+const nav = document.querySelector('.js-nav');
 
+let counter = 0;
+let maxScroll = 0;
+const screenSize = parseInt(screenBox.offsetWidth);
+const position = parseInt(opinionBox.offsetWidth)
+
+function checkScreen(screenSize) {
+    if(screenSize < 768) {
+        maxScroll = 5;
+    } else if (screenSize < 1256) {
+        maxScroll = 5;
+    } else {
+        maxScroll = 3;
+    }
+}
```

Comment:
Tę funkcję możesz wrzucić do event listenera, window może przyjmować addEventListener o wratości 'resize' który jest wywoływany podczas zmiany rozmiaru ekranu. Pamiętaj też o tym, aby sprawdzić czy uruchamia się to na początku otwarcia strony bez resize.

Po otwarciu strony => uruchamiasz i ustawiasz
On resize => sprawdzasz, ustawiasz

Dodatkowo możesz poczytać o czymś takim jak debounce, jest to sposób na wywoływanie funkcji tylko raz gdy kończymy robić resize, zamiast wywoływać ją co kilka pikseli kilkaset razy (np zmieniajac rozmiar z 1900 na 320 to wywola sie setki razy).